### ISSUE TITLE
Constanly disconnecting my calls because I'm pressing the power button

### ISSUE THREAD

**Issue by rbieb** [wireapp/wire-ios#652]
Can you please disable the "power button press disconnects the call" functionality? It's driving me crazy.

Following situation:
- I'm in a call
- I put the call on speakerphones
- I want to turn off my display
- I press the power button
- The call disconnects

The *only* way to actually disconnect a call should be to press the red "disconnect call" button. The power button should not have any effect on the call whatsoever. The standard apple phone app doesn't do it either, so why should wire?

**Comment by c-g-owen** [comment]
It seems that this is a feature of CallKit & iOS. I just tested with the normal phone app and it did the same, so I guess it is intended behaviour.

One way to get the behaviour you want is to disable CallKit in Wire app settings: "Settings(cog symbol)->Options->Calls, Share with iOS" then close and restart the app. This made it not
end the call for me.

Hope this helps

**Comment by 0ki** [comment]
For me the call disconnects only on the second time I press the button. This is intended behaviour.

**Comment by jenskasemets** [comment]
@rbieb Thank you for your comment, as mentioned above, this is iOS (10) behaviour and also regular (GSM) calls act in similar way. Nothing we can or want to change here.

**Comment by rbieb** [comment]
> this is iOS (10) behaviour and also regular (GSM) calls act in similar way

Sorry to have to correct you,  but this is not the case. A "normal" call on ios, will never be disconnected by pressing the power button. A wire call does. I've tested it again yesterday and as I've said, I've multiple times accidentally disconnected a call because I've pressed the power button, when I actually intendet to just turn off the screen.

I stand by what I said earlier. It's not behaving the same way as a normal call on ios, it's confusing and irritating to the user and it should be changed.
